### PR TITLE
DOCS-2409: Adds tiers breaking change info to CC 20.3.0 RN

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
CC @apardeshi-tigera

**Comment by ctauchen** [comment]
@mazdakn I've moved over a few changes from the CE docs to CC. In your PR for the tiers work, you sandardized these pages across product. But some details and later edits didn't make it in to CC. 

For the image changes, I'm happy to use the CE images. We really shouldn't be duplicating these.

Please take a look and let me know if any of this omissions were on purpose.

File: calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx [hunk 1]
```diff
@@ -52,9 +52,9 @@ kubectl apply -f security.yaml
 
 ## The default tier:
 
-The AdminNetworkPolicy tier where all [Kubernetes admin network policies](https://network-policy-api.sigs.k8s.io/reference/examples/) reside. It is automatically created during installation and has the order of 1,000. This value is fixed, and cannot be changed.
+The default tier is created during installation and has the order of 1,000,000. This value is fixed, and cannot be changed. The desire is to evaluate policies in the default tier as last.
```
Comment (mazdakn) [inline calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx:55]: @ctauchen oops! This was definitely a mistake. Thanks for fixing it.

File: calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx [hunk 2]
```diff
@@ -64,11 +64,11 @@ The default tier is where:
 
 ## AdminNetworkPolicy tier:
 
-The AdminNetworkPolicy tier is where all [Kubernetes admin network policies](https://network-policy-api.sigs.k8s.io/reference/examples/) reside. It is automatically created during installation and has the order of 1,000.
+The AdminNetworkPolicy tier is where all [Kubernetes admin network policies](https://network-policy-api.sigs.k8s.io/reference/examples/) reside. It is automatically created during installation and has the order of 1,000. This value is fixed, and cannot be changed.
```
Comment (mazdakn) [inline calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx:67]: @ctauchen Can you please apply the same change to Calico open source and enterprise. I think we should try to keep them in sync if possible.

Comment (ctauchen) [reply calico-cloud_versioned_docs/version-20-2/network-policy/policy-tiers/tiered-policy.mdx:67]: I think the only mistake was leaving this out of CC. I took this copy from the CE docs. Thanks!

**Pull request merged**